### PR TITLE
[RFC] xfce4-4.14

### PR DESCRIPTION
**Pull request by pbui**
This updates the core xfce4 packages to 4.14pre1 status as outlined here:

https://wiki.xfce.org/releng/4.14/xfce_4.14pre1

This also includes some revbumps and updates to packages that depend on xfconf (which has a SO bump).

I'm not sure if we want to update all the packages now, wait until 4.14 is released, or update individual packages on a case-by-case basis.  That is what I am marking this with [RFC].

### CODE REVIEW

**Comment by pbui** [comment]
Updates:

- Cross-building should work (tested against armv7l)
- Travis fails due to unrelated change:

```
libmagic-5.37_1 (update) breaks installed pkg `file-5.36_1'
```

**Comment by pbui** [comment]
Updated to [xfce-4.14pre2](https://wiki.xfce.org/releng/4.14/xfce_4.14pre2)

**Comment by pbui** [comment]
Updated for the final release of xfce4-4.14:

https://www.xfce.org/about/news/?post=1565568000

File: srcpkgs/orage/template [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Template file for 'orage'
 pkgname=orage
 version=4.12.1
-revision=3
+revision=4
 build_style=gnu-configure
 hostmakedepends="pkg-config intltool flex"
-makedepends="libical-devel libnotify-devel gtk+-devel dbus-glib-devel
+makedepends="libical-devel libnotify-devel gtk+-devel
```
Comment (xtraeme) [inline srcpkgs/orage/template]: gtk+ is gtk-2.x

Comment (pbui) [reply srcpkgs/orage/template]: Good catch, I've updated this.

**Comment by pbui** [comment]
@xtraeme Packages should be updated now... sorry about that, didn't realise they bumped between the pre3 and final release.

**Pull request merged**